app/javascript/components/Pages/ProductivityCheckLow.js

```diff
@@ -51,7 +51,7 @@ const ProductivityCheckLow = ({data, setData, saveDataToDb, steps, service}) =>
   
   
   const handlingOnClickNext = () => {
-    if (productivity <= 2) {
+    if (productivity < 3) {
       steps.push('productivity-bad-follow-up');
```

Comment:
@SerhiiBorozenets I noticed that you're making changes to the code that is already in the master branch and working properly. Could you please explain the reason for these changes?

Reply:
@LyubovP it is the same but more compact

Reply:
@SerhiiBorozenets Please ensure that these changes do not affect the functionality of ProductivityCheckLow and do not violate the overall logic. Since QA during testing will be focused on the CausesToCelebrate task, if the changes you make do not relate to this task, it may negatively affect the functionality of the program and lead to the appearance of errors in the production. Please consider this when making changes to the code that is already in the master branch and not related to the current task, to prevent possible errors in production in the future.